packages/overlay/src/ActiveOverlay.ts

```diff
@@ -362,7 +362,17 @@ export class ActiveOverlay extends SpectrumElement {
 
     private stealOverlayContent(element: HTMLElement): void {
         this.originalPlacement = element.getAttribute('placement') as Placement;
-        this.restoreContent = reparentChildren([element], this);
+        this.restoreContent = reparentChildren(
+            [element],
+            this,
+            (el: Element) => {
+                const slotName = el.slot;
+                el.removeAttribute('slot');
+                return (el: Element) => {
+                    el.slot = slotName;
+                };
+            }
```

Comment:
Does typescript automatically assign this to a single-entry array?

Reply:
Assign which part to a single entry array?

Reply:
ahh -- I just misread the parameters here (was looking at `restoreChildren` not `reparentChildren`. Now that I'm reading this right this looks great.